onnxruntime: 1.22.0 -> 1.22.2

GaetanLepage opened the pull request:
## Things done

- **[python3Packages.]onnxruntime: 1.22.0 -> 1.22.2**
  Diff: https://github.com/microsoft/onnxruntime/compare/v1.22.0...v1.22.2
  Changelog: https://github.com/microsoft/onnxruntime/releases/tag/v1.22.2
- **python3Packages.onnx: 1.18.0 -> 1.19.0**
  Diff: https://github.com/onnx/onnx/compare/v1.18.0...v1.19.0
  Changelog: https://github.com/onnx/onnx/releases/tag/v1.19.0

cc @puffnfresh @ck3d @cbourjau @acairncross 


- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
onnxruntime: 1.22.0 -> 1.22.2

Diff: https://github.com/microsoft/onnxruntime/compare/v1.22.0...v1.22.2

Changelog: https://github.com/microsoft/onnxruntime/releases/tag/v1.22.2

GaetanLepage committed:
python3Packages.onnx: 1.18.0 -> 1.19.0

Diff: https://github.com/onnx/onnx/compare/v1.18.0...v1.19.0

Changelog: https://github.com/onnx/onnx/releases/tag/v1.19.0

GaetanLepage committed:
python3Packages.onnxruntime-tools: cleanup

GaetanLepage committed:
python3Packages.onnxslim: 0.1.62 -> 0.1.68

Changelog: https://github.com/inisis/OnnxSlim/releases/tag/v0.1.68

LunNova commented:
`onnxruntime` fails for me with

```
        > 86% tests passed, 1 tests failed out of 7
┃        >
┃        > Total Test time (real) =   8.32 sec
┃        >
┃        > The following tests FAILED:
┃        >        1 - onnxruntime_test_all (ILLEGAL)
┃        > Errors while running CTest
┃        > make: *** [Makefile:71: test] Error 8
```

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 437307`
Commit: `7a0d3990baafa250881d9d5df1b606af5d8c579f`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.sparsezoo</li>
  </ul>
</details>
<details>
  <summary>:x: 31 packages failed to build:</summary>
  <ul>
    <li>docling-serve (python313Packages.docling-serve)</li>
    <li>docling-serve.dist (python313Packages.docling-serve.dist)</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>local-ai</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
    <li>python312Packages.docling-serve</li>
    <li>python312Packages.docling-serve.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.onnxconverter-common</li>
    <li>python312Packages.onnxconverter-common.dist</li>
    <li>python312Packages.onnxmltools</li>
    <li>python312Packages.onnxmltools.dist</li>
    <li>python312Packages.pytorch-pfn-extras</li>
    <li>python312Packages.pytorch-pfn-extras.dist</li>
    <li>python312Packages.skl2onnx</li>
    <li>python312Packages.skl2onnx.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.onnxconverter-common</li>
    <li>python313Packages.onnxconverter-common.dist</li>
    <li>python313Packages.onnxmltools</li>
    <li>python313Packages.onnxmltools.dist</li>
    <li>python313Packages.pytorch-pfn-extras</li>
    <li>python313Packages.pytorch-pfn-extras.dist</li>
    <li>python313Packages.skl2onnx</li>
    <li>python313Packages.skl2onnx.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 259 packages built:</summary>
  <ul>
    <li>adoptopenjdk-icedtea-web</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aitrack</li>
    <li>alpaca</li>
    <li>calibre</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>docling (python313Packages.docling)</li>
    <li>docling.dist (python313Packages.docling.dist)</li>
    <li>envision</li>
    <li>exo</li>
    <li>exo.dist</li>
    <li>firefox-beta-unwrapped</li>
    <li>firefox-beta-unwrapped.debug</li>
    <li>firefox-beta-unwrapped.symbols</li>
    <li>firefox-devedition-unwrapped</li>
    <li>firefox-devedition-unwrapped.debug</li>
    <li>firefox-devedition-unwrapped.symbols</li>
    <li>firefox-unwrapped</li>
    <li>firefox-unwrapped.debug</li>
    <li>firefox-unwrapped.symbols</li>
    <li>firefoxpwa</li>
    <li>frigate</li>
    <li>home-assistant-component-tests.tami4</li>
    <li>ieda</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>librewolf-unwrapped</li>
    <li>librewolf-unwrapped.debug</li>
    <li>livecaptions</li>
    <li>maa-assistant-arknights</li>
    <li>maa-cli</li>
    <li>magika (python313Packages.magika)</li>
    <li>magika.dist (python313Packages.magika.dist)</li>
    <li>monado</li>
    <li>newelle</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>onnxruntime</li>
    <li>onnxruntime.dev</li>
    <li>onnxruntime.dist</li>
    <li>openseeface</li>
    <li>opentrack</li>
    <li>owocr</li>
    <li>owocr.dist</li>
    <li>piper-phonemize</li>
    <li>piper-tts</li>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.brax</li>
    <li>python312Packages.brax.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.deepface</li>
    <li>python312Packages.deepface.dist</li>
    <li>python312Packages.dm-sonnet</li>
    <li>python312Packages.dm-sonnet.dist</li>
    <li>python312Packages.docling</li>
    <li>python312Packages.docling-jobkit</li>
    <li>python312Packages.docling-jobkit.dist</li>
    <li>python312Packages.docling.dist</li>
    <li>python312Packages.edward</li>
    <li>python312Packages.edward.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.faster-whisper</li>
    <li>python312Packages.faster-whisper.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.gliner</li>
    <li>python312Packages.gliner.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.keras</li>
    <li>python312Packages.keras.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llm-sentence-transformers</li>
    <li>python312Packages.llm-sentence-transformers.dist</li>
    <li>python312Packages.magika</li>
    <li>python312Packages.magika.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.markitdown</li>
    <li>python312Packages.markitdown.dist</li>
    <li>python312Packages.mhcflurry</li>
    <li>python312Packages.mhcflurry.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mtcnn</li>
    <li>python312Packages.mtcnn.dist</li>
    <li>python312Packages.onnx</li>
    <li>python312Packages.onnx.dist</li>
    <li>python312Packages.onnxruntime</li>
    <li>python312Packages.onnxruntime-tools</li>
    <li>python312Packages.onnxruntime-tools.dist</li>
    <li>python312Packages.onnxruntime.dist</li>
    <li>python312Packages.onnxslim</li>
    <li>python312Packages.onnxslim.dist</li>
    <li>python312Packages.paddle2onnx</li>
    <li>python312Packages.paddle2onnx.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.piper-phonemize</li>
    <li>python312Packages.piper-phonemize.dist</li>
    <li>python312Packages.pypasser</li>
    <li>python312Packages.pypasser.dist</li>
    <li>python312Packages.pysilero-vad</li>
    <li>python312Packages.pysilero-vad.dist</li>
    <li>python312Packages.rapidocr-onnxruntime</li>
    <li>python312Packages.rapidocr-onnxruntime.dist</li>
    <li>python312Packages.rembg</li>
    <li>python312Packages.rembg.dist</li>
    <li>python312Packages.retinaface</li>
    <li>python312Packages.retinaface.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.speechrecognition</li>
    <li>python312Packages.speechrecognition.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.tami4edgeapi</li>
    <li>python312Packages.tami4edgeapi.dist</li>
    <li>python312Packages.tf2onnx</li>
    <li>python312Packages.tf2onnx.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.tinygrad</li>
    <li>python312Packages.tinygrad.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>python312Packages.tyro</li>
    <li>python312Packages.tyro.dist</li>
    <li>python312Packages.ultralytics</li>
    <li>python312Packages.ultralytics.dist</li>
    <li>python312Packages.unsloth</li>
    <li>python312Packages.unsloth-zoo</li>
    <li>python312Packages.unsloth-zoo.dist</li>
    <li>python312Packages.unsloth.dist</li>
    <li>python312Packages.unstructured-inference</li>
    <li>python312Packages.unstructured-inference.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.docling-jobkit</li>
    <li>python313Packages.docling-jobkit.dist</li>
    <li>python313Packages.fastembed</li>
    <li>python313Packages.fastembed.dist</li>
    <li>python313Packages.faster-whisper</li>
    <li>python313Packages.faster-whisper.dist</li>
    <li>python313Packages.gliner</li>
    <li>python313Packages.gliner.dist</li>
    <li>python313Packages.insightface</li>
    <li>python313Packages.insightface.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.manifest-ml</li>
    <li>python313Packages.manifest-ml.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.onnx</li>
    <li>python313Packages.onnx.dist</li>
    <li>python313Packages.onnxruntime</li>
    <li>python313Packages.onnxruntime-tools</li>
    <li>python313Packages.onnxruntime-tools.dist</li>
    <li>python313Packages.onnxruntime.dist</li>
    <li>python313Packages.onnxslim</li>
    <li>python313Packages.onnxslim.dist</li>
    <li>python313Packages.piper-phonemize</li>
    <li>python313Packages.piper-phonemize.dist</li>
    <li>python313Packages.pypasser</li>
    <li>python313Packages.pypasser.dist</li>
    <li>python313Packages.pysilero-vad</li>
    <li>python313Packages.pysilero-vad.dist</li>
    <li>python313Packages.rapidocr-onnxruntime</li>
    <li>python313Packages.rapidocr-onnxruntime.dist</li>
    <li>python313Packages.rembg</li>
    <li>python313Packages.rembg.dist</li>
    <li>python313Packages.sentence-transformers</li>
    <li>python313Packages.sentence-transformers.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.tami4edgeapi</li>
    <li>python313Packages.tami4edgeapi.dist</li>
    <li>python313Packages.tinygrad</li>
    <li>python313Packages.tinygrad.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.ultralytics</li>
    <li>python313Packages.ultralytics.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>rapidraw</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>rocmPackages.migraphx (rocmPackages_6.migraphx)</li>
    <li>rocmPackages.mivisionx (rocmPackages.mivisionx-hip, rocmPackages_6.mivisionx, rocmPackages_6.mivisionx-hip)</li>
    <li>rocmPackages.mivisionx-cpu (rocmPackages_6.mivisionx-cpu)</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>sitespeed-io</li>
    <li>thunderbird-unwrapped (thunderbird-latest-unwrapped, thunderbirdPackages.thunderbird, thunderbirdPackages.thunderbird-latest)</li>
    <li>thunderbird-unwrapped.debug (thunderbird-latest-unwrapped.debug, thunderbirdPackages.thunderbird-latest.debug, thunderbirdPackages.thunderbird.debug)</li>
    <li>thunderbird-unwrapped.symbols (thunderbird-latest-unwrapped.symbols, thunderbirdPackages.thunderbird-latest.symbols, thunderbirdPackages.thunderbird.symbols)</li>
    <li>unbook</li>
    <li>unstructured-api</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wivrn</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>docling-serve</summary>
<pre>adding &#x27;docling_serve/datamodel/responses.py&#x27;
adding &#x27;docling_serve-1.1.0.dist-info/licenses/LICENSE&#x27;
adding &#x27;docling_serve-1.1.0.dist-info/METADATA&#x27;
adding &#x27;docling_serve-1.1.0.dist-info/WHEEL&#x27;
adding &#x27;docling_serve-1.1.0.dist-info/entry_points.txt&#x27;
adding &#x27;docling_serve-1.1.0.dist-info/top_level.txt&#x27;
adding &#x27;docling_serve-1.1.0.dist-info/RECORD&#x27;
removing build/bdist.linux-x86_64/wheel
Successfully built docling_serve-1.1.0-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/docling_serve-1.1.0...OK
Repacking wheel as ./docling_serve-1.1.0-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for docling_serve-1.1.0-py3-none-any.whl
  - docling-mcp not installed</pre>
</details>
<details>
<summary>open-webui</summary>
<pre>no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built open_webui-0.6.25-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/open_webui-0.6.25...OK
Repacking wheel as ./open_webui-0.6.25-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for open_webui-0.6.25-py3-none-any.whl
  - rapidocr-onnxruntime not installed</pre>
</details>
<details>
<summary>python312Packages.cleanlab</summary>
<pre>ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_underperforming_group_availability_issue_1065[keys=[&#x27;pred_probs&#x27;, &#x27;features&#x27;, &#x27;cluster_ids&#x27;] -&gt; available] - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_underperforming_group_availability_issue_1065[keys=[&#x27;pred_probs&#x27;, &#x27;knn_graph&#x27;, &#x27;cluster_ids&#x27;] -&gt; available] - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_underperforming_group_availability_issue_1065[keys=[&#x27;pred_probs&#x27;, &#x27;features&#x27;, &#x27;knn_graph&#x27;, &#x27;cluster_ids&#x27;] -&gt; available] - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_underperforming_group_availability_issue_1065[keys=[&#x27;pred_probs&#x27;] -&gt; unavailable] - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_get_available_issue_types - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_find_issues - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestIssueFinder::test_validate_issue_types_dict - AssertionError
ERROR tests/datalab/test_issue_finder.py::TestRegressionIssueFinder::test_get_available_issue_types - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_init - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_report - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_get_report[True] - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_get_report[False] - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_show_all_issues[True-Here is a lab summary\n\nfoo report\n\n\nbar report] - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_show_all_issues[False-Here is a lab summary\n\nfoo report] - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_summary_with_score - AssertionError
ERROR tests/datalab/test_report.py::TestReporter::test_summary_without_score - AssertionError
ERROR tests/spurious_correlation/test_spurious_correlation.py::TestImagelabReporterAdapter::test_report[dark] - AssertionError
ERROR tests/spurious_correlation/test_spurious_correlation.py::TestImagelabReporterAdapter::test_report[odd_aspect_ratio] - AssertionError
ERROR tests/spurious_correlation/test_spurious_correlation.py::TestImagelabReporterAdapter::test_report[identity] - AssertionError
= 151 failed, 652 passed, 15 deselected, 2 xfailed, 6210 warnings, 233 errors in 340.26s (0:05:40) =</pre>
</details>
<details>
<summary>python312Packages.onnxconverter-common</summary>
<pre>Executing pythonImportsCheckPhase
Check whether the following modules can be imported: onnxconverter_common
Traceback (most recent call last):
  File &quot;&lt;string&gt;&quot;, line 1, in &lt;module&gt;
  File &quot;&lt;string&gt;&quot;, line 1, in &lt;lambda&gt;
  File &quot;/nix/store/18l19gnl8dq75v6knghbcgkrikmpc0yv-python3-3.12.11/lib/python3.12/importlib/__init__.py&quot;, line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1387, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1360, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1331, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 935, in _load_unlocked
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 999, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 488, in _call_with_frames_removed
  File &quot;/nix/store/a5qjldm4qlrb99n5iii4yf7la59209dr-python3.12-onnxconverter-common-1.15.0/lib/python3.12/site-packages/onnxconverter_common/__init__.py&quot;, line 20, in &lt;module&gt;
    from .onnx_ops import *  # noqa F403
    ^^^^^^^^^^^^^^^^^^^^^^^
  File &quot;/nix/store/a5qjldm4qlrb99n5iii4yf7la59209dr-python3.12-onnxconverter-common-1.15.0/lib/python3.12/site-packages/onnxconverter_common/onnx_ops.py&quot;, line 12, in &lt;module&gt;
    from onnx.mapping import NP_TYPE_TO_TENSOR_TYPE
ModuleNotFoundError: No module named &#x27;onnx.mapping&#x27;</pre>
</details>
<details>
<summary>python312Packages.pytorch-pfn-extras</summary>
<pre>    def state_dict(self) -&gt; Dict[str, Any]:
        self._add_deferred_values()
        state = {}
        try:
            # Save the stats as python scalars in order to avoid
            # different device errors when loading them back
            state = {
&gt;               &quot;_x&quot;: float(self._x),
                      ^^^^^^^^^^^^^^
                &quot;_x2&quot;: float(self._x2),
                &quot;_n&quot;: int(self._n),
            }
E           UserWarning: Converting a tensor with requires_grad=True to a scalar may lead to unexpected behavior.
E           Consider using tensor.detach() first. (Triggered internally at /build/pytorch/torch/csrc/autograd/generated/python_variable_methods.cpp:835.)

pytorch_pfn_extras/reporting.py:360: UserWarning
=========================== short test summary info ============================
FAILED tests/pytorch_pfn_extras_tests/test_reporter.py::test_serialize_tensor_with_grad - UserWarning: Converting a tensor with requires_grad=True to a scalar may lead to unexpected behavior.
Consider using tensor.detach() first. (Triggered internally at /build/pytorch/torch/csrc/autograd/generated/python_variable_methods.cpp:835.)
========= 1 failed, 2019 passed, 65 skipped, 193 deselected in 51.23s ==========</pre>
</details>
<details>
<summary>python313Packages.onnxconverter-common</summary>
<pre>    import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
                                  ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File &quot;&lt;string&gt;&quot;, line 1, in &lt;lambda&gt;
    import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
                                                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File &quot;/nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/importlib/__init__.py&quot;, line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1387, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1360, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1331, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 935, in _load_unlocked
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 1026, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 488, in _call_with_frames_removed
  File &quot;/nix/store/5r6vyqcgclmnirjs7b5y269qk37rgkv5-python3.13-onnxconverter-common-1.15.0/lib/python3.13/site-packages/onnxconverter_common/__init__.py&quot;, line 20, in &lt;module&gt;
    from .onnx_ops import *  # noqa F403
    ^^^^^^^^^^^^^^^^^^^^^^^
  File &quot;/nix/store/5r6vyqcgclmnirjs7b5y269qk37rgkv5-python3.13-onnxconverter-common-1.15.0/lib/python3.13/site-packages/onnxconverter_common/onnx_ops.py&quot;, line 12, in &lt;module&gt;
    from onnx.mapping import NP_TYPE_TO_TENSOR_TYPE
ModuleNotFoundError: No module named &#x27;onnx.mapping&#x27;</pre>
</details>
<details>
<summary>python313Packages.pytorch-pfn-extras</summary>
<pre>    def state_dict(self) -&gt; Dict[str, Any]:
        self._add_deferred_values()
        state = {}
        try:
            # Save the stats as python scalars in order to avoid
            # different device errors when loading them back
            state = {
&gt;               &quot;_x&quot;: float(self._x),
                      ^^^^^^^^^^^^^^
                &quot;_x2&quot;: float(self._x2),
                &quot;_n&quot;: int(self._n),
            }
E           UserWarning: Converting a tensor with requires_grad=True to a scalar may lead to unexpected behavior.
E           Consider using tensor.detach() first. (Triggered internally at /build/pytorch/torch/csrc/autograd/generated/python_variable_methods.cpp:835.)

pytorch_pfn_extras/reporting.py:360: UserWarning
=========================== short test summary info ============================
FAILED tests/pytorch_pfn_extras_tests/test_reporter.py::test_serialize_tensor_with_grad - UserWarning: Converting a tensor with requires_grad=True to a scalar may lead to unexpected behavior.
Consider using tensor.detach() first. (Triggered internally at /build/pytorch/torch/csrc/autograd/generated/python_variable_methods.cpp:835.)
========= 1 failed, 2018 passed, 65 skipped, 194 deselected in 48.48s ==========</pre>
</details>
</details>

GaetanLepage commented:
> Any chance we can get a more recent protobuf version?
> 
> [#436579 (comment)](https://github.com/NixOS/nixpkgs/pull/436579#issuecomment-3249884599)

Done. Seems like it builds fine.

happysalada commented on pkgs/by-name/on/onnxruntime/package.nix:
```diff
@@ -268,7 +263,8 @@ effectiveStdenv.mkDerivation rec {
   passthru = {
     inherit cudaSupport cudaPackages; # for the python module
-    protobuf = protobuf_21;
+    # protobuf = protobuf_21;
```
Im guessing you arent done here, otherwise is there a reason why you wanted to comment out rather than remove this line ?

GaetanLepage committed:
python3Packages.onnxruntime: remove commented dependencies

GaetanLepage committed:
python3Packages.onnxconverter-common: 1.15.0 -> 0.16.0

Diff: https://github.com/microsoft/onnxconverter-common/compare/v1.15.0...v0.16.0

Changelog: https://github.com/microsoft/onnxconverter-common/releases/tag/v0.16.0

mweinelt commented:
That's what I was hoping for to happen before merge. Thanks.

prusnak commented:
Removing PR from the merge queue then

GaetanLepage committed:
python3Packages.crewai: 0.175.0 -> 0.186.1

Diff: https://github.com/crewAIInc/crewAI/compare/0.175.0...0.186.1

Changelog: https://github.com/crewAIInc/crewAI/releases/tag/0.186.1

GaetanLepage commented:
Once #443272 is merged, I guess the review should be all green!

LunNova commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 437307`
Commit: `f68891f1c05930ef02b561c9ac93bce34dc3c1e2`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.cleanlab</li>
    <li>python312Packages.cleanlab.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 7 packages failed to build:</summary>
  <ul>
    <li>firefox-beta-unwrapped</li>
    <li>firefox-beta-unwrapped.debug</li>
    <li>firefox-beta-unwrapped.symbols</li>
    <li>firefox-devedition-unwrapped</li>
    <li>firefox-devedition-unwrapped.debug</li>
    <li>firefox-devedition-unwrapped.symbols</li>
    <li>frigate</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 298 packages built:</summary>
  <ul>
    <li>adoptopenjdk-icedtea-web</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aitrack</li>
    <li>alpaca</li>
    <li>calibre</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>docling (python313Packages.docling)</li>
    <li>docling-serve (python313Packages.docling-serve)</li>
    <li>docling-serve.dist (python313Packages.docling-serve.dist)</li>
    <li>docling.dist (python313Packages.docling.dist)</li>
    <li>envision</li>
    <li>exo</li>
    <li>exo.dist</li>
    <li>firefox-unwrapped</li>
    <li>firefox-unwrapped.debug</li>
    <li>firefox-unwrapped.symbols</li>
    <li>firefoxpwa</li>
    <li>home-assistant-component-tests.tami4</li>
    <li>ieda</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>librewolf-unwrapped</li>
    <li>librewolf-unwrapped.debug</li>
    <li>livecaptions</li>
    <li>maa-assistant-arknights</li>
    <li>maa-cli</li>
    <li>magika (python313Packages.magika)</li>
    <li>magika.dist (python313Packages.magika.dist)</li>
    <li>monado</li>
    <li>newelle</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>onnxruntime</li>
    <li>onnxruntime.dev</li>
    <li>onnxruntime.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>openseeface</li>
    <li>opentrack</li>
    <li>owocr</li>
    <li>owocr.dist</li>
    <li>piper-phonemize</li>
    <li>piper-tts</li>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.brax</li>
    <li>python312Packages.brax.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.crewai</li>
    <li>python312Packages.crewai.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.deepface</li>
    <li>python312Packages.deepface.dist</li>
    <li>python312Packages.dm-sonnet</li>
    <li>python312Packages.dm-sonnet.dist</li>
    <li>python312Packages.docling</li>
    <li>python312Packages.docling-jobkit</li>
    <li>python312Packages.docling-jobkit.dist</li>
    <li>python312Packages.docling-mcp</li>
    <li>python312Packages.docling-mcp.dist</li>
    <li>python312Packages.docling-serve</li>
    <li>python312Packages.docling-serve.dist</li>
    <li>python312Packages.docling.dist</li>
    <li>python312Packages.edward</li>
    <li>python312Packages.edward.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.fastembed</li>
    <li>python312Packages.fastembed.dist</li>
    <li>python312Packages.faster-whisper</li>
    <li>python312Packages.faster-whisper.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.gliner</li>
    <li>python312Packages.gliner.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.keras</li>
    <li>python312Packages.keras.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-readers-docling</li>
    <li>python312Packages.llama-index-readers-docling.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llm-sentence-transformers</li>
    <li>python312Packages.llm-sentence-transformers.dist</li>
    <li>python312Packages.magika</li>
    <li>python312Packages.magika.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.markitdown</li>
    <li>python312Packages.markitdown.dist</li>
    <li>python312Packages.mhcflurry</li>
    <li>python312Packages.mhcflurry.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mtcnn</li>
    <li>python312Packages.mtcnn.dist</li>
    <li>python312Packages.onnx</li>
    <li>python312Packages.onnx.dist</li>
    <li>python312Packages.onnxconverter-common</li>
    <li>python312Packages.onnxconverter-common.dist</li>
    <li>python312Packages.onnxmltools</li>
    <li>python312Packages.onnxmltools.dist</li>
    <li>python312Packages.onnxruntime</li>
    <li>python312Packages.onnxruntime-tools</li>
    <li>python312Packages.onnxruntime-tools.dist</li>
    <li>python312Packages.onnxruntime.dist</li>
    <li>python312Packages.onnxslim</li>
    <li>python312Packages.onnxslim.dist</li>
    <li>python312Packages.paddle2onnx</li>
    <li>python312Packages.paddle2onnx.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.piper-phonemize</li>
    <li>python312Packages.piper-phonemize.dist</li>
    <li>python312Packages.pypasser</li>
    <li>python312Packages.pypasser.dist</li>
    <li>python312Packages.pysilero-vad</li>
    <li>python312Packages.pysilero-vad.dist</li>
    <li>python312Packages.pytorch-pfn-extras</li>
    <li>python312Packages.pytorch-pfn-extras.dist</li>
    <li>python312Packages.rapidocr</li>
    <li>python312Packages.rapidocr-onnxruntime</li>
    <li>python312Packages.rapidocr-onnxruntime.dist</li>
    <li>python312Packages.rapidocr.dist</li>
    <li>python312Packages.rembg</li>
    <li>python312Packages.rembg.dist</li>
    <li>python312Packages.retinaface</li>
    <li>python312Packages.retinaface.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skl2onnx</li>
    <li>python312Packages.skl2onnx.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.speechrecognition</li>
    <li>python312Packages.speechrecognition.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.tami4edgeapi</li>
    <li>python312Packages.tami4edgeapi.dist</li>
    <li>python312Packages.tf2onnx</li>
    <li>python312Packages.tf2onnx.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.tinygrad</li>
    <li>python312Packages.tinygrad.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>python312Packages.tyro</li>
    <li>python312Packages.tyro.dist</li>
    <li>python312Packages.ultralytics</li>
    <li>python312Packages.ultralytics.dist</li>
    <li>python312Packages.unsloth</li>
    <li>python312Packages.unsloth-zoo</li>
    <li>python312Packages.unsloth-zoo.dist</li>
    <li>python312Packages.unsloth.dist</li>
    <li>python312Packages.unstructured-inference</li>
    <li>python312Packages.unstructured-inference.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.docling-jobkit</li>
    <li>python313Packages.docling-jobkit.dist</li>
    <li>python313Packages.docling-mcp</li>
    <li>python313Packages.docling-mcp.dist</li>
    <li>python313Packages.fastembed</li>
    <li>python313Packages.fastembed.dist</li>
    <li>python313Packages.faster-whisper</li>
    <li>python313Packages.faster-whisper.dist</li>
    <li>python313Packages.gliner</li>
    <li>python313Packages.gliner.dist</li>
    <li>python313Packages.insightface</li>
    <li>python313Packages.insightface.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-readers-docling</li>
    <li>python313Packages.llama-index-readers-docling.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.manifest-ml</li>
    <li>python313Packages.manifest-ml.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.onnx</li>
    <li>python313Packages.onnx.dist</li>
    <li>python313Packages.onnxconverter-common</li>
    <li>python313Packages.onnxconverter-common.dist</li>
    <li>python313Packages.onnxmltools</li>
    <li>python313Packages.onnxmltools.dist</li>
    <li>python313Packages.onnxruntime</li>
    <li>python313Packages.onnxruntime-tools</li>
    <li>python313Packages.onnxruntime-tools.dist</li>
    <li>python313Packages.onnxruntime.dist</li>
    <li>python313Packages.onnxslim</li>
    <li>python313Packages.onnxslim.dist</li>
    <li>python313Packages.piper-phonemize</li>
    <li>python313Packages.piper-phonemize.dist</li>
    <li>python313Packages.pypasser</li>
    <li>python313Packages.pypasser.dist</li>
    <li>python313Packages.pysilero-vad</li>
    <li>python313Packages.pysilero-vad.dist</li>
    <li>python313Packages.pytorch-pfn-extras</li>
    <li>python313Packages.pytorch-pfn-extras.dist</li>
    <li>python313Packages.rapidocr</li>
    <li>python313Packages.rapidocr-onnxruntime</li>
    <li>python313Packages.rapidocr-onnxruntime.dist</li>
    <li>python313Packages.rapidocr.dist</li>
    <li>python313Packages.rembg</li>
    <li>python313Packages.rembg.dist</li>
    <li>python313Packages.sentence-transformers</li>
    <li>python313Packages.sentence-transformers.dist</li>
    <li>python313Packages.skl2onnx</li>
    <li>python313Packages.skl2onnx.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.tami4edgeapi</li>
    <li>python313Packages.tami4edgeapi.dist</li>
    <li>python313Packages.tinygrad</li>
    <li>python313Packages.tinygrad.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.ultralytics</li>
    <li>python313Packages.ultralytics.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>rapidraw</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>rocmPackages.migraphx (rocmPackages_6.migraphx)</li>
    <li>rocmPackages.mivisionx (rocmPackages.mivisionx-hip, rocmPackages_6.mivisionx, rocmPackages_6.mivisionx-hip)</li>
    <li>rocmPackages.mivisionx-cpu (rocmPackages_6.mivisionx-cpu)</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>sitespeed-io</li>
    <li>thunderbird-unwrapped (thunderbird-latest-unwrapped, thunderbirdPackages.thunderbird, thunderbirdPackages.thunderbird-latest)</li>
    <li>thunderbird-unwrapped.debug (thunderbird-latest-unwrapped.debug, thunderbirdPackages.thunderbird-latest.debug, thunderbirdPackages.thunderbird.debug)</li>
    <li>thunderbird-unwrapped.symbols (thunderbird-latest-unwrapped.symbols, thunderbirdPackages.thunderbird-latest.symbols, thunderbirdPackages.thunderbird.symbols)</li>
    <li>unbook</li>
    <li>unstructured-api</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wivrn</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>firefox-beta-unwrapped</summary>
<pre>    |
431 |     fn lock(&amp;mut self, id: ExternalImageId, channel_index: u8, is_composited: bool) -&gt; ExternalImage&lt;&#x27;_&gt; {
    |                                                                                                     ++++

   Compiling gkrust-shared v0.1.0 (/build/firefox-143.0/toolkit/library/rust/shared)
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/parser/html&#x27;
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/netwerk/ipc&#x27;
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/netwerk/protocol/http&#x27;
warning: `webrender_bindings` (lib) generated 1 warning
warning: `style` (lib) generated 86 warnings
warning: `webrender` (lib) generated 18 warnings
   Compiling gkrust v0.1.0 (/build/firefox-143.0/toolkit/library/rust)
    Finished `release` profile [optimized] target(s) in 26m 24s
toolkit/library/rust/libgkrust.a
:
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/toolkit/library/rust&#x27;
make[2]: Leaving directory &#x27;/build/firefox-143.0/objdir&#x27;
make[1]: *** [/build/firefox-143.0/config/recurse.mk:34: compile] Error 2
make[1]: Leaving directory &#x27;/build/firefox-143.0/objdir&#x27;
make: *** [/build/firefox-143.0/config/rules.mk:359: default] Error 2</pre>
</details>
<details>
<summary>firefox-devedition-unwrapped</summary>
<pre>    |
431 |     fn lock(&amp;mut self, id: ExternalImageId, channel_index: u8, is_composited: bool) -&gt; ExternalImage&lt;&#x27;_&gt; {
    |                                                                                                     ++++

   Compiling gkrust-shared v0.1.0 (/build/firefox-143.0/toolkit/library/rust/shared)
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/parser/html&#x27;
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/layout/generic&#x27;
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/netwerk/protocol/http&#x27;
warning: `webrender_bindings` (lib) generated 1 warning
warning: `style` (lib) generated 86 warnings
warning: `webrender` (lib) generated 18 warnings
   Compiling gkrust v0.1.0 (/build/firefox-143.0/toolkit/library/rust)
    Finished `release` profile [optimized] target(s) in 27m 39s
toolkit/library/rust/libgkrust.a
:
make[3]: Leaving directory &#x27;/build/firefox-143.0/objdir/toolkit/library/rust&#x27;
make[2]: Leaving directory &#x27;/build/firefox-143.0/objdir&#x27;
make[1]: *** [/build/firefox-143.0/config/recurse.mk:34: compile] Error 2
make[1]: Leaving directory &#x27;/build/firefox-143.0/objdir&#x27;
make: *** [/build/firefox-143.0/config/rules.mk:359: default] Error 2</pre>
</details>
<details>
<summary>frigate</summary>
<pre>  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/runner.py&quot;, line 344 in from_call
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/runner.py&quot;, line 391 in pytest_make_collect_report
  File &quot;/nix/store/wd72j1wywnmmq9bz6zr8g4g6m4hkxzwr-python3.12-pluggy-1.6.0/lib/python3.12/site-packages/pluggy/_callers.py&quot;, line 121 in _multicall
  File &quot;/nix/store/wd72j1wywnmmq9bz6zr8g4g6m4hkxzwr-python3.12-pluggy-1.6.0/lib/python3.12/site-packages/pluggy/_manager.py&quot;, line 120 in _hookexec
  File &quot;/nix/store/wd72j1wywnmmq9bz6zr8g4g6m4hkxzwr-python3.12-pluggy-1.6.0/lib/python3.12/site-packages/pluggy/_hooks.py&quot;, line 512 in __call__
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/runner.py&quot;, line 567 in collect_one_node
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 839 in _collect_one_node
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 974 in genitems
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 979 in genitems
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 979 in genitems
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 979 in genitems
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 979 in genitems
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 813 in perform_collect
  File &quot;/nix/store/1a5mxdmzxc50m7pml6dz3cgijy8qndv7-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/main.py&quot;, line 353 in pytest_collection
  File &quot;/nix/store/wd72j1wywnmmq9bz6zr8g4g6m4hkxzwr-python3.12-pluggy-1.6.0/lib/python3.12/site-packages/pluggy/_callers.py&quot;, line 121 in _multicall
  File &quot;/nix/store/wd72j1wywnmmq9bz6zr8g4g6m4hkxzwr-python3.12-pluggy-1.6.0/lib/python3.12/site-packages/pluggy/_manager.py&quot;, line 120 in _hookexec
  ...

Extension modules: playhouse._sqlite_ext, greenlet._greenlet, zope.interface._zope_interface_coptimizations, gevent.libev.corecext, gevent._gevent_c_greenlet_primitives, gevent._gevent_c_hub_local, gevent._gevent_c_waiter, gevent._gevent_c_hub_primitives, gevent._gevent_c_ident, gevent._gevent_cgreenlet, gevent._gevent_c_abstract_linkable, gevent._gevent_cevent, gevent._gevent_clocal, gevent._gevent_cqueue, _brotli, charset_normalizer.md, requests.packages.charset_normalizer.md, requests.packages.chardet.md, _ruamel_yaml, markupsafe._speedups, _cffi_backend, numpy._core._multiarray_umath, numpy.linalg._umath_linalg, google._upb._message, h5py._errors, h5py.defs, h5py._objects, h5py.h5, h5py.utils, h5py.h5t, h5py.h5s, h5py.h5ac, h5py.h5p, h5py.h5r, h5py._npystrings, h5py._proxy, h5py._conv, h5py.h5z, h5py.h5a, h5py.h5d, h5py.h5ds, h5py.h5g, h5py.h5i, h5py.h5o, h5py.h5f, h5py.h5fd, h5py.h5pl, h5py.h5l, h5py._selector, scipy._lib._ccallback_c, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._pcg64, numpy.random._mt19937, numpy.random._generator, numpy.random._philox, numpy.random._sfc64, numpy.random.mtrand, scipy.sparse._sparsetools, _csparsetools, _cyutility, scipy._cyutility, scipy.sparse._csparsetools, PIL._imaging (total: 65)
/nix/store/g39sc31im41rn8zmsa76yrpjiqkxg1sw-pytest-check-hook/nix-support/setup-hook: line 23:   537 Segmentation fault         (core dumped) /nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/bin/python3.12 &quot;${flagsArray[@]}&quot;</pre>
</details>
</details>

mweinelt commented:
Due to nss 3.116 update, which was reverted.
